icu4c/source/data/cldr-icu-readme.txt

```diff
@@ -188,6 +195,11 @@ make check 2>&1 | tee /tmp/icu4c-oldData-makeCheck.txt
 # a dtd cache and are still having timeout problems, the IP address of your system
 # may have been blocked due to previous excessive access. In this case you may need
 # to contact a Unicode sysadmin to restore access.
+#
+# Unfortunately, even if you have your DTD cache variable enabled, you may still
+# get blocked and unable to populate your cache because of multiple successive
+# requests to download the required DTD files. It is recommended that you
+# pre-populate your cache as shown above in step 1d.
 
```

Comment:
I don't think that happens if you have used mkdir to first create the cldrdtd directory.

Reply:
It does, though; I have my dir created, and I got one of the DTD files downloaded, and unicode.org blocked me on the second one.  Fredrik has had a similar problem.

Reply:
Hmm. I have not encountered that before.